### PR TITLE
Scatter various menu items

### PR DESCRIPTION
**Pull request by Paul-Licameli**
Resolves: 

Some dependencies that block some future module extractions are broken;
to do that, more menu items are registered.

So this is mostly just boring cutting and pasting.

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: src/Printing.cpp [hunk 1]
```diff
@@ -191,3 +195,39 @@ void HandlePrint(
...
+{
+   auto &project = context.project;
+   auto name = project.GetProjectName();
+   auto &tracks = TrackList::Get( project );
+   auto &window = GetProjectFrame( project );
+   HandlePrint(&window, name, &tracks, TrackPanel::Get( project ));
+}
+
+using namespace MenuTable;
+BaseItemSharedPtr PrintingItems()
```
Comment (ksoze95) [inline src/Printing.cpp:216]: I think could be made static

File: src/toolbars/TranscriptionToolBar.cpp [hunk 2]
```diff
@@ -1065,3 +1065,110 @@ AttachedToolBarMenuItem sAttachment{
...
+      Menu( wxT("PlayAtSpeed"), XXO("&Play-at-Speed") ) };
+   return menu;
+}
+
+AttachedItem sAttachment2{
+   wxT("Optional/Extra/Part1"),
+   Shared( ExtraPlayAtSpeedMenu() )
+};
+
+BaseItemSharedPtr ExtraPlayAtSpeedItems()
```
Comment (ksoze95) [inline src/toolbars/TranscriptionToolBar.cpp:1147]: This submenu has moved from Extra->Play-At-Speed to now Extra->Play-At-Speed->Play-At-Speed.  Is this intentional?

Comment (Paul-Licameli) [reply src/toolbars/TranscriptionToolBar.cpp:1147]: Actually this is not a mistake:  the sub-menu, and then the items within the sub-menu, are registered separatedly, each with a correct path, and when I run the program, the menu is correct.

The code might be simpler with one registration, but it is not wrong.

Comment (ksoze95) [reply src/toolbars/TranscriptionToolBar.cpp:1147]: Apologies, it looks like that I might have cut and paste (with a bit of further fixup) incorrectly applied the code transformations for the commit on my local machine.

**Review by ksoze95 — APPROVED** [review]

**Pull request merged**